setup.cfg

```diff
@@ -1,7 +1,4 @@
 [flake8]
-ignore =
-    E203
-    W503
```

Comment:
We could introduce these options to align with Optuna's flake8 options.
https://github.com/optuna/optuna/blob/11dca397313eca84dd183e57351c33f33b05e4f6/setup.cfg#L2-L5